--- origsrc/libopenspc-0.3.99/examples/Makefile.am	2005-07-23 23:12:37.000000000 -0500
+++ src/libopenspc-0.3.99/examples/Makefile.am	2008-07-02 15:18:37.703125000 -0500
@@ -3,5 +3,5 @@
 bin_PROGRAMS = ospcplay
 noinst_PROGRAMS = spccomm
 
-INCLUDES = -I../libopenspc
+INCLUDES = -I$(top_srcdir)/libopenspc
 LDADD = $(top_builddir)/libopenspc/libopenspc.la
--- origsrc/libopenspc-0.3.99/libopenspc/Makefile.am	2005-07-23 23:12:36.000000000 -0500
+++ src/libopenspc-0.3.99/libopenspc/Makefile.am	2008-07-02 15:18:05.312500000 -0500
@@ -1,11 +1,11 @@
 SUBDIRS = SNEeSe
 
 AM_CFLAGS = -W -Wall -pedantic -Wno-long-long
-INCLUDES  = -ISNEeSe
+INCLUDES  = -I$(srcdir)/SNEeSe
 
 lib_LTLIBRARIES = libopenspc.la
 libopenspc_la_LIBADD = -lm -lz SNEeSe/libSPCimpl.la
 libopenspc_la_SOURCES = main.c dsp.c dsp.h gauss.h 
-libopenspc_la_LDFLAGS = -version-info 0:3
+libopenspc_la_LDFLAGS = -version-info 0:3 -no-undefined
 
 include_HEADERS = openspc.h
--- origsrc/libopenspc-0.3.99/libopenspc/SNEeSe/Makefile.am	2005-09-26 18:38:22.000000000 -0500
+++ src/libopenspc-0.3.99/libopenspc/SNEeSe/Makefile.am	2008-07-02 15:18:21.109375000 -0500
@@ -1,5 +1,5 @@
 AM_CFLAGS = -W -Wall -pedantic -Wno-long-long
-INCLUDES  = -I..
+INCLUDES  = -I$(top_srcdir)/libopenspc
 
 noinst_LTLIBRARIES = libSPCimpl.la
 
